contracts/metagov/DelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../core/TribeRoles.sol";
+import "../pcv/PCVDeposit.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+/// @title Delegator PCV Deposit
+/// This contract simply holds an ERC20 token, and delegate its voting power
+/// to an address. The ERC20 token needs to implement a delegate(address) method.
+/// @author eswak
+contract DelegatorPCVDeposit is PCVDeposit {
+    using SafeERC20 for IERC20;
+
+    event DelegateUpdate(address indexed oldDelegate, address indexed newDelegate);
+
+    /// @notice the token that is being used for voting
+    ERC20Votes public token;
```

Comment:
actually it is set in `VlAuraDelegatorPCVDeposit` because the address of token was not known at time of deploy (and it has an initialize() function to set the state variables)